README.md

```diff
@@ -9,7 +9,7 @@
 `sshcode` is a CLI to automatically install and run [code-server](https://github.com/cdr/code-server) over SSH.
 
 It uploads your extensions and settings automatically, so you can seamlessly use
-remote servers as VS Code hosts.
```

Comment:
I think it would be cleaner if we just hyperlinked `VS Code`

Reply:
I can agree with that. Changing it now.